src/Evolution/Systems/RelativisticEuler/Valencia/Fluxes.hpp

```diff
@@ -0,0 +1,50 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace gsl {
```

Comment:
Do we need to `/// \cond` the forward decls so they don't show up in Doxygen? This is what I've been doing while running IWYU because Doxygen can be frustrating.

Reply:
what do you mean by don't show up in Doxygen?

Reply:
I've sometimes had Doxygen show forward declarations as new entities in the documentation, even though we want the definition with the documentation to show up.

Reply:
Okay, I've understood what Doxygen does. In some cases we document the partial specialization of a class template but not the declaration. The rule is we should put class template declarations in `/// \cond`

Reply:
ping